assets/translations/en-us.yaml

```diff
@@ -593,14 +614,26 @@ catalog:
     namespaceIsInProject: "This chart's target namespace, <code>{namespace}</code>, already exists and cannot be added to a different project."
     project: Install into Project
     section:
-      appReadme: README
       chartOptions: Chart Options
-      helm: Helm Deploy Options
-      readme: Helm README
       valuesYaml: Values YAML
+    steps:
+      basics:
+        label: Metadata
+        subtext: Set App metadata
+        description: This process will help you {action} your chosen chart.
+      helmValues:
+        label: Values
+        subtext: Change how the App runs
+        chartInfo:
+          button: View Chart Info
+          label: Chart Info
+      helmCli:
+        label: Command
+        subtext: Change how the Helm command runs
+        description: Rancher will {action} your chart using a CLI tool. Here's the settings you can apply for it's command.
```

Comment:
`it's` -> `its`.

This is not technically correct, helm is used as a library and not invoked as a CLI.

Though not particularly consistently applied elsewhere `Rancher` as should not be hardcoded because white-labeling will want to override it; use `{appName}` (which i18n makes [always available](https://github.com/rancher/dashboard/blob/master/store/i18n.js#L95)) unless there's a reason it must use the literal (e.g. the CLI binary download is always `rancher` because that's what it is).

Reply:
Typo and {appName} fixed. 

In terms of the description it's not great, though I thought even under the hood we use the cli (the log outputs the full helm cli command)?

Reply:
Err, sorry I was just fixing some other places and you actually want `{vendor}` if you want it to say "Rancher";  `{appName}` is "Dashboard" (which I'm switching to say Cluster Explorer because Dashboard is not a term we use in marketing/docs).

I think you could just say something more generic like "Configure Helm options for how {vendor} will install this chart."

Reply:
Ok, for the three steps i now have 
```
description: This process will help {action, select,
    install { create }
    upgrade { upgrade }
    update { update }
  } the {existing, select,
    true { app }
    false { chart }
  }. Start by setting some basic information used by {vendor} to manage the App.
```
```
description: Configure Values used by Helm to define the App.
```
```
description: Configure Options used by {vendor} to {action} the App.
```